<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qnx::Internal::BarDescriptorEditorGeneralWidget</class>
 <widget class="QWidget" name="Qnx::Internal::BarDescriptorEditorGeneralWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Orientation:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="orientation"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>Chrome:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="chrome"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="transparentMainWindow">
     <property name="text">
      <string>Transparent main window</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Application Arguments:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="applicationArguments"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
